lang/en/tool_lockstats.php

```diff
@@ -46,12 +46,14 @@
 $string['h1_slowest'] = 'Slowest tasks this week with a duration > {$a} seconds';
 $string['h1_nexttask'] = 'Next running tasks';
 $string['lock_in_use'] = 'Lock still in use by another process';
+$string['nostalelocks'] = 'No stale process locks detected.';
```

Comment:
here as well would be good to mention that not locks older than 24 hours

Reply:
good thought, I will add it